### PR TITLE
Outdated documentation setting up a test app

### LINKED ISSUE #3481: [DOCS] Outdated documentation setting up a test app
The documentation located in `/docs/sources/tempo/setup/set-up-test-app.md` is outdated and does not work as it's described in the docs.
I would suggest to fix the `use-opentelemetry-telemetrygen-to-generate-tracing-data` section and in the first step reference the `telemetrygen` repository but include a template of a Pod for generating traces

```
apiVersion: batch/v1
kind: CronJob
metadata:
  name: sample-traces
spec:
  concurrencyPolicy: Forbid
  successfulJobsHistoryLimit: 1
  failedJobsHistoryLimit: 2
  schedule: "0 4 * * *"
  jobTemplate:
    spec:
      backoffLimit: 0
      ttlSecondsAfterFinished: 3600
      template:
        spec:
          containers:
          - name: traces
            image: ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.96.0
            args:
              - traces
              - --otlp-insecure
              - --rate
              - "10"
              - --duration
              - 300s
              - --otlp-endpoint
              - grafana-agent.grafana-agent.svc.cluster.local:4317
          restartPolicy: Never
```

The documentation is not updated as the parameter `--otlp-endpoint` is not set and the user would appreciate the example to be run within Kubernetes, like for instance a Pod, Job or CronJob template.

### PR DESCRIPTION
**Pull request by ausias-armesto**
**What this PR does**:
Updates documentation on how to generate traces

**Which issue(s) this PR fixes**:
Fixes #3481

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/setup/set-up-test-app.md [hunk 1]
```diff
@@ -124,20 +124,52 @@ Replace these appropriately if you have altered the endpoint targets for the fol
...
   ```
 
-If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, eg:
-```bash
-kubectl port-forward services/grafana-agent 4317:4317 --namespace grafana-agent
+  If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, eg:
+  ```bash
+  kubectl port-forward services/grafana-agent 4317:4317 --namespace grafana-agent
+  ```
+3. Alertnatively a cronjob can be created to send traces periodically based on this template
```
Comment (joe-elliott) [inline docs/sources/tempo/setup/set-up-test-app.md:142]:
```suggestion
3. Alternatively a cronjob can be created to send traces periodically based on this template
```

**Review by joe-elliott — COMMENTED** [review]
Thanks for the improvements. Small spelling correct and it should be good!

File: docs/sources/tempo/setup/set-up-test-app.md [hunk 2]
```diff
@@ -124,20 +124,52 @@ Replace these appropriately if you have altered the endpoint targets for the fol
...
   ```
 
-If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, eg:
-```bash
-kubectl port-forward services/grafana-agent 4317:4317 --namespace grafana-agent
+  If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, eg:
+  ```bash
+  kubectl port-forward services/grafana-agent 4317:4317 --namespace grafana-agent
+  ```
+3. Alternatively a cronjob can be created to send traces periodically based on this template
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/set-up-test-app.md:142]:
```suggestion
3. Alternatively, a cronjob can be created to send traces periodically based on this template:
```

File: docs/sources/tempo/setup/set-up-test-app.md [hunk 3]
```diff
@@ -124,20 +124,52 @@ Replace these appropriately if you have altered the endpoint targets for the fol
...
   Optionally, you can also send the trace directly to the Tempo database without using Grafana Agent as a collector by using the following:
   ```bash
-  telemetrygen traces --otlp-insecure --rate 20 --duration 5s tempo-cluster-distributor.tempo.svc.cluster.local:4317
+  telemetrygen traces --otlp-insecure --rate 20 --duration 5s --otlp-endpoint tempo-cluster-distributor.tempo.svc.cluster.local:4317
   ```
 
-If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, eg:
-```bash
-kubectl port-forward services/grafana-agent 4317:4317 --namespace grafana-agent
+  If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, eg:
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/set-up-test-app.md:138]:
```suggestion
  If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, for example:
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the doc! There are some small suggestions, but that's it.

File: docs/sources/tempo/setup/set-up-test-app.md [hunk 4]
```diff
@@ -124,20 +124,52 @@ Replace these appropriately if you have altered the endpoint targets for the fol
 1. Install `telemetrygen` using the [installation procedure](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/telemetrygen).
    **NOTE**: You don't need to configure an OpenTelemetry Collector as we are using the Grafana Agent.
 
-1. Generate traces using `telemtrygen`:
+2. Generate traces using `telemtrygen`:
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/set-up-test-app.md:127]:
```suggestion
2. Generate traces using `telemetrygen`:
```

**Pull request merged**